### PR TITLE
Port in element tests

### PR DESCRIPTION
**Pull request by piotrpalek**
**DONE**

Related issue: #561

### CODE REVIEW

**Comment by piotrpalek** [comment]
@chadhietala would appreciate a quick review to see if I'm going in the right direction with this

**Comment by chadhietala** [comment]
Looking good. For quotes what you've done is fine.

File: packages/@glimmer/runtime/test/in-element-test.ts [hunk 1]
```diff
@@ -1,487 +1,441 @@
...
-    }
-  );
-
-  equalsElement(externalElement, 'div', {}, stripTight``);
-  assert.equal(destroyed, 0, 'component was destroyed');
-
-  set(view, 'showExternal', true);
-  rerender();
-
-  assertElementIsEmberishElement(externalElement.firstElementChild!, 'div', { }, 'destroy me!');
```
Comment (piotrpalek) [inline packages/@glimmer/runtime/test/in-element-test.ts]: @chadhietala not sure what to do about this, it doesn't seem to render the same way?

Comment (chadhietala) [inline packages/@glimmer/runtime/test/in-element-test.ts]: Not sure if I follow? Isn't this tested on L428 of the new tests?

Comment (piotrpalek) [inline packages/@glimmer/runtime/test/in-element-test.ts]: @chadhietala sorry that probably wasn't very clear. What I meant is that I couldn't use the `assertElementIsEmberishElement` method in the new tests and instead just did  `equalsElement`. My question was if this is an issue or if it's fine.

**Comment by piotrpalek** [comment]
@chadhietala other than the one question I had it should be good to go if it looks good to you

File: packages/@glimmer/runtime/test/in-element-test.ts (hunk 1 above)
Comment (chadhietala) [inline packages/@glimmer/runtime/test/in-element-test.ts]: This seems fine.

**Pull request merged**